test/live_view_native/jetpack/rules_parser_test.exs

```diff
@@ -197,7 +225,7 @@ defmodule LiveViewNative.Jetpack.RulesParserTest do
     end
 
     test "parses key/value pairs" do
-      input = ~s|foo(bar: "baz", qux: .quux)|
+      input = ~s|foo(bar = "baz", qux = .quux)|
```

Comment:
@bcardarella, why do we need to support `=` for key value pairs? I can't figure out the pattern for when `:` is used over `=`

Reply:
@NduatiK in Kotlin this is the syntax

Reply:
@nelson-glauber can you confirm that we always want `key = value` instead of `key: value` ?

Reply:
If it's for the sake of conformity with Kotlin, then no worries.